test/controllers/UploadControllerSpec.scala

```diff
@@ -467,11 +484,85 @@ class UploadControllerSpec extends FrontEndTestHelper {
     }
   }
 
+  "UploadController addFileStatus" should {
+    "call the addFileStatus endpoint" in {
+      val graphQLConfiguration: GraphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val client = graphQLConfiguration.getClient[addFileStatus.Data, addFileStatus.Variables]()
+      val consignmentService: ConsignmentService = new ConsignmentService(graphQLConfiguration)
+      val fileStatusService = new FileStatusService(new GraphQLConfiguration(app.configuration))
```

Comment:
You can reuse the `graphQLConfiguration` variable you've got here instead of calling `new GraphQLConfiguration(app.configuration)`
It's the same for the other two tests as well.

Reply:
Sorry, I missed this. I have updated it, thanks.